manifests/plugin/remote_execution/ssh.pp

```diff
@@ -54,40 +54,44 @@
     listen_on => $listen_on,
   }
 
-  if $ssh_kerberos_auth {
-    $kerberos_pkg = "${foreman_proxy::params::ruby_package_prefix}net-ssh-krb"
-    package { $kerberos_pkg:
-      ensure => present,
+  if $mode =~ /ssh/ {
+    if $ssh_kerberos_auth {
+      $kerberos_pkg = "${foreman_proxy::params::ruby_package_prefix}net-ssh-krb"
```

Comment:
`net-ssh-krb` is not required anymore since `smart_proxy_remote execution_ssh-0.5.0`

Reply:
Removed it.

If preferred, I can separate that part into another commit for better logical clarity.

Reply:
I'd even prefer it as a separate PR since we should already merge it.

Reply:
https://github.com/theforeman/puppet-foreman_proxy/pull/732

Since it's no longer packaged in foreman repositories, I made the change to `ensure => purged`